http/src/isolate.ts

```diff
@@ -44,9 +44,10 @@ export function isolateSink(
       if (typeof req === 'string') {
         return {url: req, _namespace: [scope]} as RequestOptions;
       }
-      req._namespace = req._namespace || [];
-      req._namespace.unshift(scope);
-      return req;
+      return {
+        ...req,
+        _namespace: [scope, ...(req._namespace || [])]
```

Comment:
It doesn't matter much in this case, but just out of curiosity, do we have the right version of TypeScript that supports `??` to replace `||`? (I suppose updating TypeScript is a subtask for the next version of Cycle)

Reply:
No, the version in the current version is too old. For the rewrite PR it is currently already on the newest version